execution_workflows/LABRAT/README.md

```diff
@@ -0,0 +1,11 @@
+# APAeval nextflow pilot run
```

Comment:
Seems to be for the pilot and QAPA. Could you adapt Readme to labrat?

Reply:
Thank you for the suggestions! I have updated the README specifically for LABRAT